content/the-ember-times-issue-194.md

```diff
@@ -44,6 +44,19 @@ In [another episode](https://www.whiskeywebandwhatnot.fm/robbies-origin-story-le
 
 ---
 
+## [Ember Ignite virtual mini conference 🔥](TODO)
+
+Ember Ignite is an international virtual mini conference scheduled for Wednesday, February 23rd!
+
+Many teams talk about the challenges of hiring and training Ember developers, and this conference tackles those challenges directly.
+Attendees will level up their training and teaching skills in order to build strong teams.
+
+Tickets are donation-based with a suggested amount of $30 USD. Proceeds will go towards creating an open source curriculum for teaching Ember to junior developers and taking a live cohort of 10 junior engineers (with no experience) and teaching them Ember so well, that they'll go toe-to-toe against mid-level UI engineer candidates. This effort is led by [Jai Bhagat](https://twitter.com/ChaiWithJai).
+
+[Register here](TODO) today!
```

Comment:
The event link is:  https://www.eventbrite.com/e/ember-ignite-tickets-232969998537

I think with your suggestions (fully incorporated) re:  landing page copy, the description is much more clear.

Reply:
Added the link!